compiler/rustc_lint_defs/src/builtin.rs

```diff
@@ -178,8 +178,9 @@ declare_lint! {
     Warn,
     "applying forbid to lint-groups",
     @future_incompatible = FutureIncompatibleInfo {
-        reason: FutureIncompatibilityReason::FutureReleaseErrorReportInDeps,
+        reason: FutureIncompatibilityReason::FutureReleaseError,
```

Comment:
```suggestion
        reason: FutureIncompatibilityReason::FutureReleaseErrorReportInDeps,
```

I'm sure the CI will catch up, but I think this is the variant name now.